src/main/rx/rx.c

```diff
@@ -846,6 +856,11 @@ uint16_t rxGetLinkQuality(void)
     return linkQuality;
 }
 
+uint8_t rxGetRFmode(void)
```

Comment:
Should also be bounded with `#ifdef USE_RX_LINK_QUALITY_INFO`.

Reply:
It's already part of the #ifdef that starts a few lines above

Reply:
Ok, sorry. Couldn't see that in the diff snippet.